devel/gk/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gk
+version                 1.0.6
+
+categories              devel
+platforms               darwin
+license                 CC-BY-3.0-US
+installs_libs           no
+maintainers             sergiolms \
+                        alejandroSuch \
+                        JBUinfo
+
+PortGroup               github 1.0
+
+github.setup            gitkraken gk-cli ${version} v
+github.tarball_from     releases
+supported_archs         x86_64 arm64
+
+
+description             Official CLI for GitKraken
+long_description        ${name} is GitKraken on the command line. It makes working across multiple \
+                        repos easier with Workspaces, provides access to pull requests and \
+                        issues from multiple services (GitHub, GitLab, Bitbucket, etc.), \
+                        and seamlessly connects with GitKraken Client and GitLens in VS Code \
+                        to visualize git information when you need it.
+homepage                https://github.com/gitkraken/gk-cli
```

Comment:
`homepage` is also implicitly set by `github.setup`.  This line does not need to be specified.